ios/Capacitor/Capacitor/Plugins/Browser.swift

```diff
@@ -23,7 +23,7 @@ public class CAPBrowserPlugin : CAPPlugin, SFSafariViewControllerDelegate {
         self.vc = SFSafariViewController.init(url: url!)
         self.vc!.delegate = self
         let presentationStyle = call.getString("presentationStyle")
-        if presentationStyle != nil && presentationStyle == "popover" {
+        if presentationStyle != nil && presentationStyle == "popover" && UIDevice.current.userInterfaceIdiom == .pad {
```

Comment:
does this mean that popover presentation on iPhone is gone? is there a reason for this? i liked the iOS13 modal-style of it.